pkg/errors/errors.go

```diff
@@ -0,0 +1,51 @@
+package errors
+
+import (
+	"errors"
+	"fmt"
+)
+
+type wrappedError struct {
+	msg  string
+	errs []error
+}
+
+// Error returns the stored error string
+func (we wrappedError) Error() string {
+	return we.msg
+}
+
+func (we wrappedError) Is(err error) bool {
+	for _, e := range we.errs {
+		if errors.Is(e, err) {
+			return true
+		}
+	}
+	return false
+}
+
+// Simple error wrapper that chains errors and prints them in a "%s(: %s)*" format
+func NewError(message string, err error, more ...error) error {
```

Comment:
pls use `github.com/hashicorp/go-multierror` if it is possible

Reply:
It seems I don't need this wrapper at all, I want to error.ErrorIs to work in the case that I have a base error and I want to wrap it to add some additional information, but it seems `fmt.Errorf` works if I use `%w`.